### ISSUE TITLE
Attempting connection to an SFTP before a VPN connection is established causes issues 'disconnecting'

### ISSUE THREAD

**Issue by timprogrammer** [iterate-ch/cyberduck#14325]
So I often have to connect to an SFTP over a VPN... Sometimes I forget to connect it before clicking connect in my bookmarks

So double click bookmark.. it'll sit trying to connect
Connect VPN
It'll still sit trying to connect
double click bookmark again to 'disconnect' (really i just want to retry connecting)
it'll sit there disconnecting (when it hasn't even connected) for a lot longer than my patience :-)

You have to close and reopen window to get things back in line and be able to double click and connect

**Comment by dkocher** [comment]
This depends on _Preferences → Connection → Timeout_ how long the initial connection attempt takes until failure. There might be a problem that it is not possible to interrupt the initial connection attempt.

**Comment by timprogrammer** [comment]
Yeah I think that's probably a better description - I cannot interrupt the connection / disconnect it while it is still attempting

**Comment by AiAe** [comment]
I have similar problem but when I turn off the VPN before I click disconnect, even with the fix @dkocher suggested, it cannot go lower than 10 seconds.
